src/tutorials/first_game_br.md

```diff
@@ -0,0 +1,166 @@
+---
+id: first_game_br
+name: First Game BR
+title: Build Your First Battle Royale!
+tags:
+    - Tutorial
+---
+
+# Create an Battle Royale in Core
+
+## Overview
+
+The Battle Royale framework features a game where the player must skydive to a starting location, the play area shrinks over time, and the last player standing wins. It will also include random weapon spawns using a loot table.
+
+## Create a Framework Game
+
+To get started, use the **CREATE** menu to make a new project
+
+### Open the **NEW PROJECT** Menu
+
+![Create New](../img/NewFrameworks/NewFrameworks_NewProject.png){: .center loading="lazy" }
+
+1. With Core open, click the **CREATE** tab in the top menu bar.
+2. Click the **NEW PROJECT** tab at the top of the **CREATE** menu.
+
+### Select the **Battle Royale** Framework
+
+<!--- TODO image of BR framework choice --->
+
+1. In the **GAMEPLAY FRAMEWORKS** section, select **Battle Royale**.
+2. In the right side panel, name your project.
+3. Click **CREATE**.
+
+## Modify the Airplane Template
+
+The game starts with the players inside of a paper airplane. You can modify the appearance of the airplane by updating the **Helper Plane** template.
+
+<!--- TODO add image of default airplane --->
+
+### Deinstance the Original Template
+
+1. In the **Project Content** window, search for the `Helper Plane` template.
+2. Drag the template into the **Hierarchy**.
+3. Right click the template and select **Deinstance this Template**
+
+<!--- TODO add image of hierarchy --->
+
+### Change the Colors
+
+Navigate to the **Helper_Plane > ClientContext > Geo** group in the **Hierarchy**. There are three components that make up the airplane. One at a time, select a component and change the **Color Override** property inside the **Properties** window.
+
+<!--- TODO add image of Color Override property --->
+
+### Add a Decal
+
+You can also add more components to the airplane. Decals can be used to display a graphic on a surface.
```

Comment:
More context. Continue customization. "Components" tends to mean something more functional.

I do love that you're teaching decals here.